projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-toolbar-dropdown/dropdown-content.tsx

```diff
@@ -191,12 +194,16 @@ type AiAssistantToolbarDropdownContentProps = {
  */
 export default function AiAssistantToolbarDropdownContent( {
 	blockType,
+	clientId,
 	disabled = false,
 	onAskAiAssistant,
 	onRequestSuggestion,
 }: AiAssistantToolbarDropdownContentProps ): ReactElement {
 	const blockQuickActions = quickActionsList[ blockType ] ?? [];
 
+	const { getBlockParents } = select( 'core/block-editor' );
+	const blockParents = getBlockParents( clientId );
```

Comment:
The other option we can do is to always display the transform option and just find the root list and transform the entire thing every time, regardless of where they click AI Assistant.

Reply:
This sounds interesting, do you think is doable without refactoring too much? Can we explore it? UX wise I think it'd be much more comfortable for the user